content/en/tracing/guide/configuring-primary-operation.md

```diff
@@ -0,0 +1,73 @@
+---
+title: Primary Operations in Services
+kind: guide
+---
+
+## Services in APM
+
+APM Services calculate trace metrics for errors, throughput and latency are calculated based on resources that match a single span name, deemed the primary operation. These service metrics are used throughout the product, both as the default Service Page, in the Service List, and the Service Map. 
+
+** Note: Trace Metrics can be queried based on their `trace.*` [namespace][1]. 
+
+## Defining a Primary Operation for a Service
+
+The primary operation name of a service determines how a service is represented in the UI. Our backend automatically selects an operation name that is deemed the entry-point into the service based on the throughput of requests. 
+
+As an example, a `web-store` service can have multiple endpoints which will be instrumented as resources. These resources will then share the same primary operation  as the entry-point into these resources is consistent, i.e. the resources `/user/home` and `/user/new` should both have the same primary operation `web.request`. In different languages a primary operation for a service may look like:
```

Comment:
```suggestion
As an example, a `web-store` service can have multiple endpoints which are instrumented as resources. These resources then share the same primary operation because the entry-point into these resources is consistent. For example, the resources `/user/home` and `/user/new` should both have the same primary operation `web.request`. In different languages a primary operation for a service may look like:
```